access/ms-teams/Makefile

```diff
@@ -0,0 +1,77 @@
+VERSION=10.0.2
+GO_VERSION=1.18.4
```

Comment:
Don't forget to update `go.mod` as well, or just rebase from `master`

Reply:
Done